### DISCUSSION TITLE
tag viewer

### DISCUSSION THREAD

**Discussion by irysova** [AprilSylph/XKit-Rewritten#257] [Suggestion Box]
Like a button at the bottom of the post which would show various tags other users put under a post. 
I missed this feature so much ever since it randomly stopped working on the other xkit lol

**Comment by invalidCards** [comment]
This is part of native Tumblr now - click on the notes of any post on your dashboard, and then click the hashtag in the top-right corner.
![image](https://user-images.githubusercontent.com/842080/132906817-2b945cb0-14b7-4ee2-88f1-6097207ff9b6.png)

**Reply by irysova** [reply]
omg thanks
